src/bitops.c

```diff
@@ -16,6 +16,36 @@
 /* Count number of bits set in the binary array pointed by 's' and long
  * 'count' bytes. The implementation of this function is required to
  * work with an input string length up to 512 MB or more (server.proto_max_bulk_len) */
+#if defined(__x86_64__) && defined(__GNUC__)
```

Comment:
why it can't support non-x86?
it seems that gcc and clang are all support __builtin_popcountll().

Reply:
I'm not sure whether it can make performance boost on ARM or RISC-V,  i add macro just for safety. Is it a little bit aggressive to instead all origin algorithm?

Reply:
yes, we need more benchmark on all the architectures, we can replace the origin algorithm if it make performance boost in all architectures.

Reply:
I would trust GCC to use the fastest method. If the compiler is new enough to use __builtin_popcountll then it will use an instruction if possible. If not then it will use some optimized algorithm.